### PR TITLE
Updating 1.22 schedule with final deadlines

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Add one of the following kinds:

/kind documentation

#### What this PR does / why we need it:
Updating 1.22 schedule with final deadlines

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:
None

/cc @annajung @divya-mohan0209 @erismaster @kikisdeliveryservice @pmmalinov01 @Pensu

### CODE REVIEW

File: releases/release-1.22/README.md [hunk 1]
```diff
@@ -72,11 +72,11 @@ The 1.22 release cycle is proposed as follows:
 | 1.22.0-rc.0 released | Branch Manager (@puerco) | Tue July 20 | week 13 | |
 | release-1.22 branch created | Branch Manager | Tue July 20 | week 13 | |
 | release-1.22 jobs created | Branch Manager | Tue July 20 | week 13 | |
-| Start final draft of Release Notes | Release Notes Lead | TBD | TBD | |
-| Release blog ready to review | Comms / Docs | TBD | TBD | |
+| Start final draft of Release Notes | Release Notes Lead | Tue July 20 | week 13 | |
+| Release blog ready to review (23:59 PDT) | Comms / Docs | Web July 21 | Week 13 | |
```
Comment (erismaster) [inline releases/release-1.22/README.md:76]:
```suggestion
| Release blog ready to review (23:59 PDT) | Comms / Docs | Wed July 21 | Week 13 | |
```

**Review by erismaster — CHANGES_REQUESTED** [review]

File: releases/release-1.22/README.md (hunk 1 above)
Comment (savitharaghunathan) [reply releases/release-1.22/README.md]: done

**Pull request merged**